modules/atf/util.lua

```diff
@@ -27,17 +38,29 @@ function table2str(o)
   end
   return tostring(o)
 end
+
+--- Print lua tables to console 
+-- @param in list of tables
 function print_table(t,... )
   if (type(t) == 'table' ) then
     print(table2str(t).. table2str(table.pack(...)))
   else
     print(tostring(t).. table2str(table.pack(...)))
   end
 end
+
+--- Check file exist ans by given path
+-- @param name absolute path to the file
+-- @return boll existence
 function is_file_exists(name)
   local f = io.open(name,"r")
   if f ~=nil then io.close(f) return true else return false end
 end
+
+--- Remove table filed by name and value
+-- @param t table for manipulation
+-- @param k key value for remove
+-- @return result table
 function table.removeKey(t, k)
```

Comment:
@LuxoftAKutsan,
We can manually set the type of function - global, local, method of the class (see ldoc documentation).